app/main/views.py

```diff
@@ -517,3 +520,116 @@ def mousereport_pdf(animal_id):
     stylesheets = [CSS(url_for('static', filename='styles.css')),
                    CSS(url_for('static', filename='datajoint.css'))]
     return render_pdf(HTML(string=html), stylesheets=stylesheets)
+
+
+@main.route('/surgery', methods=['GET', 'POST'])
```

Comment:
in general you should add some more comments or at least separate different tasks inside a same function with blank lines. I like to think of it like writing in paragraphs with a comment being the topic sentence
```python
# Performing taks 1
slslls;
slslssll

# Performing task 2
sjsksk
slslsls   # some more detail about this weird ass line
slsls
```